end_to_end/signal_handler_test.go

```diff
@@ -71,9 +104,14 @@ var _ = Describe("Signal handler tests", func() {
 				rng := rand.New(rand.NewSource(time.Now().UnixNano()))
 				time.Sleep(time.Duration(rng.Intn(1000)+500) * time.Millisecond)
 				_ = cmd.Process.Signal(unix.SIGINT)
+
+				for outScanner.Scan() {
```

Comment:
This fragment also repeats several times. IMHO it's better to extract it to the function.